File: src/main/java/no/nav/syfo/domain/rest/MotebehovsvarVarselInfo.java
package no.nav.syfo.domain.rest;

import lombok.Data;
import lombok.experimental.Accessors;

@Data
@Accessors(fluent = true, chain = true)
public class MotebehovsvarVarselInfo {
    public String sykmeldtAktorId;
    public String orgnummer;
}

File: src/main/kotlin/no/nav/syfo/varsel/MotebehovsvarVarselInfo.kt
package no.nav.syfo.varsel

import java.io.Serializable

data class MotebehovsvarVarselInfo(
    val sykmeldtAktorId: String,
    val orgnummer: String,
    val naermesteLederFnr: String,
    val arbeidstakerFnr: String,
) : Serializable

data class MotebehovsvarSykmeldtVarselInfo(
    val sykmeldtAktorId: String,
    val arbeidstakerFnr: String,
    val orgnummer: String?,
) : Serializable
